pyblhost/pyblhost.py

```diff
@@ -572,6 +600,7 @@ def _data_callback(self, data: bytearray) -> None:
             #     pass
             else:
                 self.logger.error("BlhostBase: Unhandled command tag: {}".format(tag))
+            self._get_command_response_event.set()
```

Comment:
Move this into `elif tag == self.ResponseTags.GetPropertyResponse:`

Reply:
See my response and new comment above.